app/code/community/Oyst/OneClick/Block/Adminhtml/Catalog/Product/Edit/Tabs.php

```diff
@@ -0,0 +1,162 @@
+<?php
+class Oyst_OneClick_Block_Adminhtml_Catalog_Product_Edit_Tabs
+    extends Mage_Adminhtml_Block_Catalog_Product_Edit_Tabs
```

Comment:
@rus-daniel @adriansavuosf I'm not fan with extending Magento file, because if a module already do that the customer have to fix conflicts.
I would like you to use this kind of approch  https://fishpig.co.uk/magento/tutorials/custom-tabs-magento-product-admin/ you can also check this module (https://fishpig.co.uk/magento/wordpress-integration/) which enable tab in product.